bin/oli/Cargo.toml

```diff
@@ -60,7 +60,7 @@ dirs = "5.0.1"
 env_logger = "0.10"
 futures = "0.3"
 log = "0.4"
-opendal.workspace = true
+opendal = {path="../../core"}
```

Comment:
Ditto https://github.com/apache/opendal/pull/4134/files#r1477020121

Reply:
The same with https://github.com/apache/opendal/pull/4134#discussion_r1477048529